lua/sql.lua

```diff
@@ -339,33 +240,34 @@ end
 ---@todo support unnamed or anonymous args
 ---@todo handle inconflict case
 function sql:insert(...)
+  local method = "insert"
   local args = {...}
   local ret_vals = {}
 
   local inner_eval = function(tbl, p)
-    return self:eval(self:parse("insert", {
-      tbl = tbl,
+    local sqlstmt = parse(tbl, method, {
       values = p,
-      placeholders = true,
-    }), p)
+      named = true,
+    })
+    return self:eval(sqlstmt, p)
   end
 
   if u.is_str(args[1]) then
     local tbl = args[1]
     local params = args[2]
-    assert_tbl(self, tbl)
+    assert_tbl(self, tbl, "insert")
```

Comment:
Just use method here

Reply:
hmmm what if you have many tables your doing a method on and you don't know which one is failing. for that we need tbl name and method